### PR TITLE
Connect API to graph on detailed data page

### PR DESCRIPTION
**Pull request by natalievolk**
Take time series data from API and use daily data to populate graph

### CODE REVIEW

File: app/app.py [hunk 1]
```diff
@@ -107,6 +114,47 @@ def get_prices(list):
...
+    return prices_dict, api_codes[item]
+
+
+def parse_data_for_graph(prices_dict, api_code):
+    labels = []
+    data = []
+
+    for date in prices_dict:
+        labels.append(date)
+        data.append(1 / prices_dict[date][api_code])
```
Comment (natalievolk) [inline app/app.py:153]: @shreya2902 Do you mind confirming that I did this correctly with taking the inverse?

Comment (shreya2902) [reply app/app.py:153]: Yes, that looks perfect!

Comment (shreya2902) [reply app/app.py:153]: Yeah, that looks perfect!

**Review by shreya2902 — APPROVED** [review]
Looks great to me!

**Review by JGSPS — APPROVED** [review]

**Pull request merged**